File: fanduel-net/Models/Player.cs
using System;

namespace fanduel_net.Models;

public class Player
{
    public string Id { get; set; }
    public int Num { get; set; }
    public string Name { get; set; }
    public Team Team { get; set; }

    public Player(string id, int num, string name, Team team)
    {
        Id = id;
        Num = num;
        Name = name;
        Team = team;
    }
}

File: fanduel-net/ClientApp/src/app/models/Player.ts
import { Team } from './Team';

export class Player {
  id: string;
  num: number;
  name: string;
  team: Team

  constructor(id: string, num: number, name: string, team: Team) {
    this.id = id;
    this.num = num;
    this.name = name;
    this.team = team;
  }
}
